### ISSUE TITLE
The menu items in the header are not differentiated from the site name.

### ISSUE THREAD

**Issue by klonos** [backdrop-contrib/basis#40]
Here's some ideas:
1. Increase the spacing between `header-identity-wrapper` and `header-menu`
2. Align menu to the right? ...not entirely sure about that. Especially if we implement something like https://github.com/vadikom/smartmenus (with the core responsive menus in https://github.com/backdrop/backdrop-issues/issues/1457) that would allow users to align left/right/center.
3. Decrease font size

Also, why are we giving so much emphasis on the "My account" and "Log out" links, aligning them along with the main menu (before it actually) and also rendering them with the same font size? Doesn't feel right.

**Comment by quicksketch** [comment]
This is sort of related to my issue about alignment of the menu links here: https://github.com/backdrop-contrib/basis/issues/39

I don't think the menu links have been styled according to the mockups yet. The image on the project page looks quite a bit better regarding the account links.

**Comment by klonos** [comment]
Fair enough. I understand that everything is WIP. Lets just keep this issue open until this is fixed though.
